<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaskDlg</class>
 <widget class="QDialog" name="MaskDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>490</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>490</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Create Mask</string>
  </property>
  <widget class="QTDisplay" name="Display1" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>471</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>191</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Draw Modes</string>
   </property>
   <widget class="QPushButton" name="includeButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Include</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boxButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>BOX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="polyButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>POLY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="excludeButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Exclude</string>
    </property>
   </widget>
   <widget class="QPushButton" name="discardButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Discard</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closepolyButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ClosePoly</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setCropButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SetCrop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ResetCrop</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>450</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddAlignModelsButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>230</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Align Model</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ClearAlignModelsButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>200</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Align Models</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTDisplay</class>
   <extends>QWidget</extends>
   <header location="global">qtdisplay.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>MaskDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
